lib/util/is-windows.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Returns if the system is a Windows system or not
+ * @returns 
+ */
+export function isWindows() {
+    return /^win/.test(globalThis.process ? globalThis.process.platform : "");
```

Comment:
is there a reason this is a function now, and not a constant? globalThis.process should never change during runtime?

Reply:
ah for the test, I see